modules/nginxvts/collect.go

```diff
@@ -0,0 +1,85 @@
+package nginxvts
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/netdata/go.d.plugin/pkg/stm"
+	"github.com/netdata/go.d.plugin/pkg/web"
+)
+
+const vtsURLPath = "/status/format/json"
+
+func (vts *NginxVTS) collect() (map[string]int64, error) {
+	ms, err := vts.scapeTotalMetrics()
+	if err != nil {
+		return nil, nil
+	}
+
+	collected := make(map[string]interface{})
+	vts.collectMainMetrics(collected, ms)
+	vts.collectSharedZonesMetrics(collected, ms)
+	vts.collectServerZonesMetrics(collected, ms)
+
+	return stm.ToMap(collected), nil
+}
+
+func (vts *NginxVTS) collectMainMetrics(collected map[string]interface{}, ms *vtsMetrics) {
+	collected["loadmsec"] = ms.LoadMsec
+	collected["nowmsec"] = ms.NowMsec
+	collected["uptime"] = ms.NowMsec - ms.LoadMsec
```

Comment:
```suggestion
	collected["uptime"] = (ms.NowMsec - ms.LoadMsec) / 1000
```

to convert to seconds